scripts/download/tools.mjs

```diff
@@ -229,6 +229,45 @@ export default async function main(platform) {
       expectedChecksum:  steveSHA,
       checksumAlgorithm: 'sha512'
     });
+
+  // Download Rancher Dashboard
+  const rancherDashboardVersion = 'v2.6.3-desktop.12';
+  const rancherDashboardURLBase = `https://github.com/rak-phillip/dashboard/releases/download/${ rancherDashboardVersion }`;
+  const rancherDashboardExecutable = 'rancher-dashboard-desktop-embed';
+  const rancherDashboardURL = `${ rancherDashboardURLBase }/${ rancherDashboardExecutable }.tar.gz`;
+  const resourcesRoot = path.join(process.cwd(), 'resources');
+  const rancherDashboardPath = path.join(resourcesRoot, 'rancher-dashboard.tgz');
```

Comment:
The `tgz` doesn't end up being ignored or deleted…

Reply:
fixed